logger.go

```diff
@@ -78,12 +82,13 @@ func (mw *MutexWrap) Disable() {
 // It's recommended to make this a global instance called `log`.
 func New() *Logger {
 	return &Logger{
-		Out:          os.Stderr,
-		Formatter:    new(TextFormatter),
-		Hooks:        make(LevelHooks),
-		Level:        InfoLevel,
-		ExitFunc:     os.Exit,
-		ReportCaller: false,
+		Out:           os.Stderr,
+		Formatter:     new(TextFormatter),
+		Hooks:         make(LevelHooks),
+		Level:         InfoLevel,
+		ExitFunc:      os.Exit,
+		ReportCaller:  false,
+		ErrorCallback: func(e error) {},
```

Comment:
Should the default hook implement current behavior of printing the error to stderr? That seems to me like the best option.